setup_wizard/character_rig_setup/rig_script.py

```diff
@@ -1,6 +1,11 @@
 # Authors: enthralpy, Llama.jpg
 # Setup Wizard Integration by michael-gh1
 
+import bpy
+import os
+from mathutils import Color, Vector
+from math import pi
+
```

Comment:
```suggestion

allow_back_move = True

```

Since the script is now inside a function and `allow_back_move` gets used as a global variable, we'll need to declare it outside of the scope of the function.

Do you see any issues with declaring it here?